src/main/java/Duke.java

```diff
@@ -1,10 +1,209 @@
+
+
+
+    public static void main(String[] args) throws DukeExceptions {
+
+        //creating list of task
+        ArrayList<Task> taskings = new ArrayList<>();
+
+        File file = new File("/Users/desmond/ip/data");
+        file.mkdir();
+        File file1 = new File("/Users/desmond/ip/data/duke.txt");
+        try {
+            file1.createNewFile();
+        } catch (IOException e) {
+            System.out.println("An error occured");
+        }
+
+        try {
+
+            File file2 = new File("/Users/desmond/ip/data/duke.txt");
+            Scanner sc = new Scanner(file2);
+            sc.useDelimiter("/");
+
+            while (sc.hasNext()) {
+
+                //Get the info for the pet
+                Task loadingTask;
+
+                String function = sc.next();
+
+
+                String done = sc.next();
+                boolean isDone = false;
+
+                if (done == "1") {
+                    isDone = true;
+                }
+
+
+                if (function == "D") {
+                    String description = sc.next();
+                    String date = sc.next();
+                    loadingTask = new Deadline (description, date, isDone);
+                    taskings.add(loadingTask);
+                } else if (function == "T") {
+                      String restOfDescription = sc.next();
+                    loadingTask = new Todo (restOfDescription,isDone);
+                    taskings.add(loadingTask);
+                } else if (function == "E") {
+                    String description = sc.next();
+                    String date = sc.next();
+                    loadingTask = new Event (description, date, isDone);
+                    taskings.add(loadingTask);
+                }
+        }
+
+            sc.close();
+
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\n");
+        System.out.println("What can I do for you?\n");
+        Scanner in = new Scanner(System.in);
+
+        //user inputs into line string
+        String line;
+
+
+
+        Task[] list;
+        list = new Task[100];
+
+        //variables for list
+        int itemNumber = 0;
+        int itemDone = 0;
+        int numeration = 1;
+        int taskNumber = 0;
+
+        //variables to check command
+        boolean startsWithBye = false;
+        boolean startsWithList = false;
+        boolean startsWithDone = false;
+        boolean startsWithTodo = false;
+        boolean startsWithDeadline = false;
+        boolean startsWithEvent = false;
+        boolean startsWithDelete = false;
```

Comment:
Consider appending these boolean variables with helping verbs to help make them sound more like a boolean instead of a statement so that it matches closer to the coding standard guide